### PR TITLE
Kubernetes documentation update

### PR DESCRIPTION
**Pull request by philip-cc**
### Description
This PR updates the Kubernetes readme to reflect the changes in #248.

### Relevant Issues/PRs
Issue #230 
PR #248 

### Type(s) of Change (Split the PR if you check many)
- [ ] Added user feature
- [ ] Added technical feature
- [ ] Added tests
- [ ] Refactor
- [ ] Bugfix
- [x] Updated documentation
- [ ] Kubernetes changes
- [ ] CI/CD workflow changes

### Testing Checklist
- [ ] I have added/updated unit tests for functional changes.
- [ ] I have added/updated integration tests for changes that affect dependent systems.
- [ ] I have added/updated end-to-end tests for feature changes.

### General Checklist
- [x] I have pulled in the latest changes from master.
- [ ] I have run `make lint`.
- [ ] I have successfully run `make tests`.
- [ ] I have updated relevant CI/CD workflows.
- [ ] I have updated relevant Kubernetes manifests/scripts.
- [x] I have updated/added relevant (internal and external) documentation (readme, doc comments, etc).
- [ ] I have updated the dependency map to reflect my changes.
- [ ] I have added the license to new source code files.
- [ ] I have spent some time looking over the full diff before creating this PR.

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -1,328 +1,184 @@
...
 ```
+Note that while the node sizes and resource requests specified in the Kubernetes files are
+sufficient to run the setup we recommend monitoring resource usage and adjust the values according
+to your use case.
 
-## Encryption Service Deployment
+In order to use the provided makefile you will need to fill out the configuration in the two
+environment files [env/deploy_env](./env/deplo_env) and [env/provider_env](./env/provider_env). Note
+in particular that you will need to create a hostname for each cluster as defined in
+[env/deploy_env](./env/deplo_env).
```
Comment (claudiu-cc) [inline kubernetes/README.md:88]: typo in the link (./env/deplo_env) -> (./env/deploy_env)

File: kubernetes/README.md [hunk 2]
```diff
@@ -1,328 +1,184 @@
...
+You can check the status with
 ```bash
-HOSTNAME=<CockroachDB Hostname>
-TLSOPTS="sslmode=verify-ca&sslrootcert=encryptonize-secrets/ca.crt&sslcert=encryptonize-secrets/client.root.crt&sslkey=encryptonize-secrets/client.root.key"
-echo 'CREATE DATABASE IF NOT EXISTS auth;' | cockroach sql --url "postgresql://root@${HOSTNAME}:26257/?${TLSOPTS}"
-cockroach sql --url "postgresql://root@${HOSTNAME}:26257/auth?${TLSOPTS}" < ../encryptonize/data/auth_storage_basic.sql
-cockroach sql --url "postgresql://root@${HOSTNAME}:26257/auth?${TLSOPTS}" < ../encryptonize/data/auth_storage_extended.sql
+kubectl -n cockroachdb get service/cockroachdb-public
 ```
+Assign the hostname you set in [env/deploy_env](./env/deplo_env) (e.g. `auth.example.com`) to the
```
Comment (claudiu-cc) [inline kubernetes/README.md:113]: Same as above

File: kubernetes/README.md [hunk 3]
```diff
@@ -1,328 +1,184 @@
...
+This will deploy Rook and wait for the Ceph cluster to be initialized.
 
-Wait until everything has started:
+After deployment, wait for an external IP to be provisioned for the `ceph-ingress` service. You can
+check the status with
 ```bash
-watch kubectl -n encryptonize get all
+kubectl -n rook-ceph get service/ceph-ingress
 ```
+Assign the hostname you set in [env/deploy_env](./env/deplo_env) (e.g. `object.example.com`) to the
```
Comment (claudiu-cc) [inline kubernetes/README.md:133]: Same as above

File: kubernetes/README.md (hunk 1 above)
Comment (claudiu-cc) [reply kubernetes/README.md:88]: This link seems to be broken everywhere in the readme

Comment (philip-cc) [reply kubernetes/README.md]: Fixed

**Review by claudiu-cc — APPROVED** [review]

**Pull request merged**